helm_deploy/demo-app/templates/deployment.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: django
+  labels:
+    app: django
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: django-container
+  template:
+    metadata:
+      labels:
+        app: django-container
+    spec:
+      containers:
+        - name: django
+          image: 926803513772.dkr.ecr.eu-west-1.amazonaws.com/cloud-platform-demo-app:latest
```

Comment:
image addr is best in values.yaml

Reply:
Yep - `helm create $chart` will automatically create a `values.yml` file containing this:

```
image:
  repository: nginx
  tag: stable
  pullPolicy: IfNotPresent
```

We should use that structure, as it's a de-facto standard at this point.